airflow/models/taskinstance.py

```diff
@@ -3892,9 +3892,17 @@ def schedule_downstream_tasks(self, session: Session = NEW_SESSION, max_tis_per_
 
         :meta: private
         """
-        return TaskInstance._schedule_downstream_tasks(
-            ti=self, session=session, max_tis_per_query=max_tis_per_query
-        )
+        try:
+            return TaskInstance._schedule_downstream_tasks(
+                ti=self, session=session, max_tis_per_query=max_tis_per_query
+            )
+        except Exception:
+            self.log.exception(
+                "Error scheduling downstream tasks. Skipping it as this is entirely optional optimisation. "
+                "There might be various reasons for it, please take a look at the stack trace to figure "
+                "out if the root cause can be diagnosed and fixed. See the issue "
+                "https://github.com/apache/airflow/issues/39717 for details and example problem."
```

Comment:
```suggestion
                "https://github.com/apache/airflow/issues/39717 for details and an example problem."
```

It may also be a good idea to tell users what to do after gettting this. Just ignore it? (What can I do if it’s noisy?) Post a comment to the issue or not?

Reply:
Sure. Good idea as well. Added some more things.